kubetoken.go

```diff
@@ -11,7 +11,7 @@ var SearchBase string = "DC=example,DC=com"
 var SearchGroups string = "kube"
 
 // NamespaceRegex is used to extract customer, namespace, and env from ldap queries
-var NamespaceRegex string = `^kube-(?P<customer>\w+)-(?P<ns>\w+)-(?P<env>\w+)-dl-`
+var NamespaceRegex string = `^kube-(?P<customer>\w+)-(?P<ns>[a-z0-9]([-a-z0-9]*[a-z0-9])?)-(?P<env>\w+)-dl-`
```

Comment:
This is the official regex for object names.
```console
k create ns bla.bla.bla
The Namespace "bla.bla.bla" is invalid: metadata.name: Invalid value: "bla.bla.bla": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```
We may want to change this to ensure the part after `-dl-` does not contain dashes to reduce ambiguity. WDYT?